AuxTel/Standard-Operations/Daytime-Operations/latiss-daily-calibrations-BIAS-DARK-FLAT-all-filters-empty-procedure.rst

```diff
@@ -43,232 +45,246 @@ Procedure Steps
 #. Enable ``Scheduler:2`` with a valid scheduler configuration. Use the standard script :file:`auxtel/scheduler/enable.py` with the configuration required for the run and available in the corresponding night log. 
 #. Run the script :file:`add_block.py` to the ATQueue  with the following configuration:
```

Comment:
I think this step could be called something like: 
**Setup LATISS calibrations** BLOCK will setup *ATCS* and white light for calibrations. It enables and turns on the *ATWhiteLight*, enables *OCPS:1* and commands AuxTel mount and dome to the FLAT position. 
Run the script :file:`add_block.py` to the ATQueue  with the following configuration:

basically what have underneath but in the first paragraph to avoid reading between lines to see what the block does.